README.md

```diff
@@ -31,8 +31,9 @@ let g:keepeye_autostart = v:true                       " launch at vim startup
 let g:keepeye_callback = 'keepeye#Callback'            " customize the callback
 let g:keepeye_message = 'SAVE YOUR EYES, TAKE A BREAK' " set the message to show
 let g:keepeye_system_bell = v:false                    " enable the bell sound (1)
+let g:keepeye_bell_player = 'mocp -l'                  " set a custom to ring the bell"
```

Comment:
I would call it `g:keepeye_system_bell_cmd`, to be more in phase with `g:keepeye_system_bell`

Reply:
And I would even split in 2 variables to be more concise: `g:keepeye_system_bell_cmd` and `g:keepeye_system_bell_cmd_flags` (and it's gonna be easier to check if it's an executable or not)

Reply:
Thanks for giving me a better name for that. I was very unhappy with the name I gave :o
keep_system_bell_cmd sounds better

I'm not sure if I go with you on this second change. It sounds like too many options where it is not needed. But it will make the code easier to understand because we don't have to split the variable to check if the binary exists or not. I vote for keeping it like this (because I believe it will be easir for the user to have only one variable to configure a custom player) but I can change it if you feel like it is better having two variables. What do you think?